cache/image_cache.go

```diff
@@ -72,22 +72,22 @@ func (c *ImageCache) RetrieveMetadata() (lifecycle.CacheMetadata, error) {
 	return meta, nil
 }
 
-func (c *ImageCache) AddLayerFile(sha string, tarPath string) error {
+func (c *ImageCache) AddLayerFile(tarPath string, diffID string) error {
```

Comment:
This seems to be a **breaking change** in the API. It's particularly scary because of how ellusive it could be given that the method signature is the same as far as types are concerned.

Reply:
It seems unlikely that github.com/buildpacks/lifecycle/cache has external consumers https://godoc.org/github.com/buildpacks/lifecycle/cache?importers